### PR TITLE
ttfautohint: update to version 1.8.3

### PR DESCRIPTION
**Pull request by lemzwerg**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.7.5 11G63
Xcode 4.6.3 4H1503 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/ttfautohint/Portfile [hunk 1]
```diff
@@ -17,8 +17,8 @@ maintainers        @tuffnatty openmaintainer
 license            {freetype GPL-2}
 master_sites       sourceforge:project/freetype/ttfautohint/${version}
 
-checksums          rmd160 74697057f5d27afb112b7b61dcc4b286fb3a9b26 \
-                   sha256 386741701596a8b2d5fb744901922ed2bd740490f7e6c81e5d7e83ac677889a7
+checksums          rmd160 67e59778818bd0c71e4eff6ca437bd994800797e \
+                   sha256 87bb4932571ad57536a7cc20b31fd15bc68cb5429977eb43d903fa61617cf87e
```
Comment (mf2k) [inline graphics/ttfautohint/Portfile]: We are now adding the `size` parameter to `checksums`.

**Comment by tuffnatty** [comment]
@lemzwerg Thank you! Still, I would wait for an official announcement of the release at https://www.freetype.org/ttfautohint/index.html. Hmm... Could it be that you are Werner, the author, BTW?

**Comment by lemzwerg** [comment]
Yes, I'm the author :-)
I'll do the announcement today evening.

File: graphics/ttfautohint/Portfile (hunk 1 above)
Comment (lemzwerg) [reply graphics/ttfautohint/Portfile]: Done.

**Comment by tuffnatty** [comment]
> Yes, I'm the author :-)

Thanks for the great product!

It looks like the CI build failure is in no way related with the patch, so OK to merge.

**Pull request merged**